platform/android/src/run_loop.cpp

```diff
@@ -55,6 +55,15 @@ int looperCallbackDefault(int fd, int, void* data) {
     return 1;
 }
 
+int looperCallbackReadEvent(int fd, int, void* data) {
+    auto runLoopImpl = reinterpret_cast<RunLoop::Impl*>(data);
+
+    if (runLoopImpl->readPoll.count(fd)) {
+        runLoopImpl->readPoll[fd](fd, RunLoop::Event::Read);
```

Comment:
assertion that the contained std::function contains valid target

Reply:
Done